packages/opencensus-core/src/stats/model/measure.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Measure, MeasureType} from './types';
+
+export class StatsMeasure implements Measure {
```

Comment:
Why does this need to be a class? It looks like you could just create plain objects of type `Measure` easily. Same comment with `StatsMeasurement.`

Reply:
This class was divided in two classes based on the measure type, like java. StatsMeasurement was removed, kept the interface.